### PR TITLE
Adds explanatory bullet points to pages 2 and 4

### PR DESCRIPTION
**Pull request by Dominik1999**
Delets tooltip from page 1 and page 7 (launch)

### CODE REVIEW

**Comment by Dominik1999** [comment]
Closes https://github.com/raiden-network/raiden-wizard/issues/213

File: resources/templates/account.html [hunk 1]
```diff
@@ -53,31 +53,19 @@
     <div class="tooltip">
       <ul>
         <li>
-          Raiden accounts are Ethereum accounts defined by a pair of keys, a
-          private key and public key.
+          Download you keystore file now. You can restore your Raiden account
```
Comment (fredo) [inline resources/templates/account.html:56]: `Please, download your keysto...`

File: resources/templates/account.html [hunk 2]
```diff
@@ -53,31 +53,19 @@
...
+          with this file and the password via Metamask.
         </li>
         <li>
-          Every private key/address pair is encoded in a keyfile. Keyfiles are
-          JSON text files which you can open and view in any text editor.
+          Send 0.125 ETH to your Raiden account. 0.075 ETH are used as safety buffer
+          to operate 3 channels. 0.05 ETH are needed for RDN and DAI tokens.
```
Comment (fredo) [inline resources/templates/account.html:61]: Here the calculation is confusing. 0.075 + 0.05 = 0.08.  Maybe it makes sense to add also information for what the rest is needed (transaction fees and buffer)

File: resources/templates/account.html [hunk 3]
```diff
@@ -53,31 +53,19 @@
...
         </li>
-        <li>You need to download and store this keyfile somewhere safe.</li>
         <li>
-          The critical component of the keyfile, your account's private key, is
-          always encrypted, and it is encrypted with the password you entered
-          when you created the account.
+          You can withdraw those 0.125 ETH if not spent in a transaction or
+          as transaction fees.
```
Comment (fredo) [inline resources/templates/account.html:65]: What do you mean by `withdraw as transaction fees`

File: resources/templates/account_password.html [hunk 4]
```diff
@@ -32,22 +32,19 @@
     <div class="tooltip">
       <ul>
         <li>
-          Raiden accounts are Ethereum accounts defined by a pair of keys, a
+          Raiden Accounts are Ethereum Accounts defined by a pair of keys, a
           private key and public key.
         </li>
         <li>
-          Every private key/address pair is encoded in a keyfile. Keyfiles are
+          Every private key/address pair is encoded in a keystore file. Keystore files are
```
Comment (fredo) [inline resources/templates/account_password.html:39]: just a nitpick: Maybe second Keystore files to `They`

File: resources/templates/account.html (hunk 2 above)
Comment (Dominik1999) [reply resources/templates/account.html:61]: 0.075 + 0.050 (=0.05) = 0.125, oder? Ich glaube transaction fees and buffer verwirrt die Leute

File: resources/templates/account.html (hunk 3 above)
Comment (Dominik1999) [reply resources/templates/account.html:65]: `          You can withdraw those 0.125 ETH minus the amount you will have spent in a 
          transaction or as transaction fees.`

besser?

File: resources/templates/account.html (hunk 2 above)
Comment (fredo) [reply resources/templates/account.html:61]: oh verrechnet
alles gut

File: resources/templates/account.html (hunk 3 above)
Comment (fredo) [reply resources/templates/account.html]: perfekt. Jetzt verstehe ich was du meinst

**Pull request merged**